task2.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');  
+const {pipeline} = require('stream');
+const csvToJsonV2 = require("csvtojson");
+
+const csvFilePath ='./csvdirectory/example.csv';
+const jsonFilePath ='./jsondirectory/test';
```

Comment:
in json directory we should create .json files

Reply:
my mistake, we need to created .txt file. I renamed it